### ISSUE TITLE
button_press and axis_move events no longer firing in Chrome 78

### ISSUE THREAD

**Issue by urlloader** [ArunMichaelDsouza/joypad.js#1]
The `button_press` and `axis_move events` are no longer firing as of Chrome 78.

I traced the cause back to the fact that `navigator.getGamepads();` no longer returns a `Gamepad[]` but a `GamepadList` instance instead in Chrome 78. 

Hence in `src/loop.js` line `14` and `15` now always evaluate to `false`. 

```js
        const gamepads = window.navigator.getGamepads();
        const { buttonEvents } = joypad;

        if (Object.keys(gamepads).length) {
            Object.keys(gamepads).forEach(index => {
        // ...
```

If was able to do a quick fix like so:

```js
        var gamepads = window.navigator.getGamepads();
        const { buttonEvents } = joypad;

        if (!Array.isArray(gamepads)) {
            gamepads = Array.prototype.slice.call(gamepads);
        }

        // Loop all the gamepads on each frame
        if (Object.keys(gamepads).length) {
            Object.keys(gamepads).forEach(index => {
        // ...
```

**Comment by Cypaubr** [comment]
I have this problem too.

**Comment by ArunMichaelDsouza** [comment]
Hi @urlloader and @Cypaubr, thanks a lot for reporting the issue. It has been fixed in [v2.0.1](https://github.com/ArunMichaelDsouza/joypad.js/releases/tag/v2.0.1).
